### ISSUE TITLE
nvim aborts with an error when open a large markdown file.

### ISSUE THREAD

**Issue by zirho** [suan/vim-instant-markdown#98]
OSX 10.11
vim versions are below.
I tried to open this file `https://github.com/zirho/WebpackTutorial/blob/master/part2/README.md`
[link to file](https://github.com/zirho/WebpackTutorial/blob/master/part2/README.md)
gives this error below.
Regular vim works fine.

When I cut some of file content, nvim opens just fine. 
Not sure what cause this. 

Thanks for the awesome plugin!

```
"README.md" 463L, 14310CAssertion failed: (!stream->closed), function stream_close, file /tmp/neovim20160302-32636-13u8lm1/neovim-0.1.2/src/nvim/event/stream.c, line 79.
[1]    54749 abort      nvim README.md
```
### nvim version

✘  /Projects/webpack/WebpackTutorial/part2   master ●  vi --version
NVIM 0.1.2
Build type: RelWithDebInfo
Compilation: /usr/local/Library/ENV/4.3/clang -Wconversion -O2 -g -DDISABLE_LOG -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wvla -fstack-protector-strong -fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -DHAVE_CONFIG_H -I/tmp/neovim20160302-32636-13u8lm1/neovim-0.1.2/build/config -I/tmp/neovim20160302-32636-13u8lm1/neovim-0.1.2/src -I/tmp/neovim20160302-32636-13u8lm1/neovim-0.1.2/deps-build/usr/include -I/tmp/neovim20160302-32636-13u8lm1/neovim-0.1.2/deps-build/usr/include -I/tmp/neovim20160302-32636-13u8lm1/neovim-0.1.2/deps-build/usr/include/luajit-2.0 -I/tmp/neovim20160302-32636-13u8lm1/neovim-0.1.2/deps-build/usr/include -I/tmp/neovim20160302-32636-13u8lm1/neovim-0.1.2/deps-build/usr/include -I/tmp/neovim20160302-32636-13u8lm1/neovim-0.1.2/deps-build/usr/include -I/tmp/neovim20160302-32636-13u8lm1/neovim-0.1.2/deps-build/usr/include -I/opt/local/include -I/usr/include -I/opt/local/include -I/tmp/neovim20160302-32636-13u8lm1/neovim-0.1.2/build/src/nvim/auto -I/tmp/neovim20160302-32636-13u8lm1/neovim-0.1.2/build/include
Compiled by zirho6@MBPro2.local

Optional features included (+) or not (-): +acl   +iconv    +jemalloc
For differences from Vim, see :help vim-differences

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/local/Cellar/neovim/0.1.2/share/nvim"
# 
### regular vim version

✘  /Projects/webpack/WebpackTutorial/part2   master ●  vim --version
VIM - Vi IMproved 7.4 (2013 Aug 10, compiled May 28 2015 21:05:17)
MacOS X (unix) version
Included patches: 1-712
Compiled by Homebrew
Huge version without GUI.  Features included (+) or not (-):
+acl             +farsi           +mouse_netterm   +syntax
+arabic          +file_in_path    +mouse_sgr       +tag_binary
+autocmd         +find_in_path    -mouse_sysmouse  +tag_old_static
-balloon_eval    +float           +mouse_urxvt     -tag_any_white
-browse          +folding         +mouse_xterm     -tcl
++builtin_terms  -footer          +multi_byte      +terminfo
+byte_offset     +fork()          +multi_lang      +termresponse
+cindent         -gettext         -mzscheme        +textobjects
-clientserver    -hangul_input    +netbeans_intg   +title
+clipboard       +iconv           +path_extra      -toolbar
+cmdline_compl   +insert_expand   +perl            +user_commands
+cmdline_hist    +jumplist        +persistent_undo +vertsplit
+cmdline_info    +keymap          +postscript      +virtualedit
+comments        +langmap         +printer         +visual
+conceal         +libcall         +profile         +visualextra
+cryptv          +linebreak       +python          +viminfo
+cscope          +lispindent      -python3         +vreplace
+cursorbind      +listcmds        +quickfix        +wildignore
+cursorshape     +localmap        +reltime         +wildmenu
+dialog_con      -lua             +rightleft       +windows
+diff            +menu            +ruby            +writebackup
+digraphs        +mksession       +scrollbind      -X11
-dnd             +modify_fname    +signs           -xfontset
-ebcdic          +mouse           +smartindent     -xim
+emacs_tags      -mouseshape      -sniff           -xsmp
+eval            +mouse_dec       +startuptime     -xterm_clipboard
+ex_extra        -mouse_gpm       +statusline      -xterm_save
+extra_search    -mouse_jsbterm   -sun_workshop    -xpm
   system vimrc file: "$VIM/vimrc"
     user vimrc file: "$HOME/.vimrc"
 2nd user vimrc file: "~/.vim/vimrc"
      user exrc file: "$HOME/.exrc"
  fall-back for $VIM: "/usr/local/share/vim"
Compilation: /usr/bin/clang -c -I. -Iproto -DHAVE_CONFIG_H   -F/usr/local/Frameworks -DMACOS_X_UNIX  -Os -w -pipe -march=native -mmacosx-version-min=10.10 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1
Linking: /usr/bin/clang   -L. -L/usr/local/lib -L/usr/local/lib -F/usr/local/Frameworks -Wl,-headerpad_max_install_names -o vim        -lm  -lncurses -liconv -framework Cocoa   -fstack-protector  -L/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -lperl -F/usr/local/Cellar/python/2.7.10/Frameworks -framework Python   -lruby.2.0.0 -lobjc

**Comment by elliotec** [comment]
Same issue here. I removed this plugin from my init.vim and i am able to open the markdown file just fine now.

**Comment by nhooyr** [comment]
In neovim/neovim#5421, @manielng fixed it by updating the python module. `pip install --upgrade neovim`.

**Comment by ashwinvis** [comment]
Should be fixed in #144
